x-pack/plugins/security_solution/public/common/components/drag_and_drop/draggable_keyboard_wrapper_hook/helpers.ts

```diff
@@ -5,7 +5,8 @@
  * 2.0.
  */
 
-import type { FluidDragActions, Position } from 'react-beautiful-dnd';
+import type { Position } from 'css-box-model';
```

Comment:
This is being imported from a library not explicitly defined as a dependency in `package.json`. Is that a concern?

Reply:
@elastic/kibana-operations, do you mind chiming in on your preference here?

Reply:
Good catch.  If it's imported directly to Kibana's code base we'll want to add it to package.json.  There could be a scenario where the nested dependency is updated and not compatible with our use.   I expect they'll resolve to the same node module so it shouldn't have an impact on bundle size.

Reply:
I'll add the new package now.